packages/clerk-js/src/ui/elements/UserAvatar.tsx

```diff
@@ -1,27 +1,26 @@
 import type { UserResource } from '@clerk/types';
 
 import { getFullName, getInitials } from '../../utils/user';
-import { useOptions } from '../contexts';
 import { Avatar } from '../elements';
 import type { PropsOfComponent } from '../styledSystem';
 
-type UserAvatarProps = PropsOfComponent<typeof Avatar> &
-  Partial<Pick<UserResource, 'firstName' | 'lastName' | 'profileImageUrl' | 'experimental_imageUrl'>> & {
+type UserAvatarProps = Omit<PropsOfComponent<typeof Avatar>, 'imageUrl'> &
+  Partial<Pick<UserResource, 'firstName' | 'lastName' | 'profileImageUrl' | 'imageUrl'>> & {
```

Comment:
☁️ if this is an internal Component i think we can omit `profileImageUrl`  from props, else i think we should leave the `profileImageUrl` but mark it as deprecated somehow.

Reply:
correct, removed